doc/examples/gatk/README.md

```diff
@@ -0,0 +1,156 @@
+# Variant Discovery with GATK
+
+![alt tag](pipeline.png)
+
+This example illustrates the use of GATK in Pachyderm for Germline variant calling and joint genotyping. Each stage of this GATK best practice pipeline can be scaled individually and is automatically triggered as data flows into the top of the pipeline. The example follows [this tutorial](https://drive.google.com/open?id=0BzI1CyccGsZiQ1BONUxfaGhZRGc) from GATK, which includes more details about the various stages.
+
+## Committing the reference genome
+
+You can retrieve all the input data for this pipeline from the Broad Institute [here](https://drive.google.com/open?id=0BzI1CyccGsZicE5HNkR6anpLTnM). We will utilize a b37 human genome reference containing only a subset of chromosome 20, which we prepared specially for GATK tutorials in order to provide a reasonable size for download. It is accompanied by its index and sequence dictionary.
+
+Download `GATK_Germline.zip` from the link above. Then:
+
+```sh
+$ unzip GATK_Germline.zip
+Archive:  GATK_Germline.zip
+   creating: data/
+  inflating: data/.DS_Store
+   creating: __MACOSX/
+   creating: __MACOSX/data/
+  inflating: __MACOSX/data/._.DS_Store
+  inflating: data/.Rhistory
+   creating: data/bams/
+  inflating: data/bams/.DS_Store
+  etc...
+$ cd data/ref
```

Comment:
instead of making users `cd`, you could also just name the file (pc put-file repo master file-name -c -f file). Whichever you think is easier for users to follow. all the cd so you dont get directories in pfs seems odd

Reply:
Yeah, I figured the `cd` would be less confusing for multiple files, especially below where they have various names and are committed in the same commit. Can adjust later if I hear any confusion.